Classes/Issues/Title/IssueTitleModel.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  IssueTitleModel.swift
+//  Freetime
+//
+//  Created by Ryan Nystrom on 1/13/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+import IGListKit
+
+final class IssueTitleModel: ListDiffable {
+
+    let attributedString: NSAttributedStringSizing
+    let trailingMetadata: Bool
```

Comment:
Can't figure out what this is for / when one would like to use it; can you clarify?

Reply:
Yup. If you don't have any "metadata" (labels, assignees, reviewers, milestone, "view files") then if we use a title bottom padding of 12 and a root-comment top padding of 24, you end up w/ an abnormally large padding between the title and root comment.

Reply:
Ah, makes sense. Just wasn't sure what the metadata could mean or be. :)